packages/core/src/login/webview/vue/login.vue

```diff
@@ -677,4 +691,25 @@ body.vscode-light #logo-text {
 .topMargin {
     margin-top: 12px;
 }
+.sign-in-help-link {
+    align-items: center;
+    display: flex;
+    font-size: 10px;
+    font-weight: 400;
+    margin: 4px 0 0;
+    text-decoration: none;
+}
+.sign-in-help-link__icon {
```

Comment:
these names are very specific. is there any opportunity to inherit common parts? else these are essentially the same as inline styles.

Reply:
The naming convention follows BEM convention - https://getbem.com/

+1. We should have a few common classes for `flex` and aligning items `center`.

Reply:
Done.